### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by hbaniecki**
#65

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ dist: trusty
 
 env:
   global:
-  - R_CHECK_ARGS="--no-build-vignettes --no-manual --timings"
+  - R_CHECK_ARGS="--no-build-vignettes --no-manual --timings --run-donttest"
```
Comment (pbiecek) [inline .travis.yml:8]: why 'run-donttest'?
wont we miss new bugs?

Comment (hbaniecki) [reply .travis.yml:8]: A lot of examples are commented in `donttest`. `--run--donttest` will make the travis check these examples.

Comment (hbaniecki) [reply .travis.yml:8]: Although i see `found \donttest examples: check also with --run-donttest` in travis so this might not work.

Comment (pbiecek) [reply .travis.yml:8]: Thanks for clarification!

**Pull request merged**